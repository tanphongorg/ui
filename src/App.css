div.containerPadding {
  padding-top: 5%;
}