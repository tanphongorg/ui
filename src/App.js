import React from 'react'
import { Container } from 'semantic-ui-react'

import Menu from './components/Menu'

import HomePage from './pages/HomePage'
import AboutPage from './pages/AboutPage'
import SearchPage from './pages/SearchPage'
import NewCoursePage from './pages/NewCoursePage'

import { Route } from 'react-router-dom'

const App = () => (
  <Container>
    <Menu />
    <Route exact path='/' key={'home' + new Date()} component={HomePage} />
    <Route path='/about' key={'about' + new Date()} component={AboutPage} />
    <Route path='/search' key={'search' + new Date()} component={SearchPage} />
    <Route path='/new_course' component={NewCoursePage} />
  </Container>
)
export default App
