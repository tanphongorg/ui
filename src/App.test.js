/* global it , expect */

import React from 'react'
import App from './App'
import renderer from 'react-test-renderer'

it('renders without crashing', () => {
  const component = renderer.create(<App />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
