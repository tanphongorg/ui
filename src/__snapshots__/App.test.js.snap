// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div>
  <MenuExampleSecondary />
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    path="/about"
  />
  <Route
    component={[Function]}
    path="/search"
  />
</div>
`;
